po/zh-TW.po

```diff
@@ -2,1151 +2,1171 @@ msgid ""
 msgid "Dangling References"
 msgstr ""
 
-#: src/SUMMARY.md:31 src/basic-syntax/slices.md:1
+#: src/SUMMARY.md:32 src/basic-syntax/slices.md:1
 msgid "Slices"
 msgstr ""
 
-#: src/SUMMARY.md:32
+#: src/SUMMARY.md:33
 msgid "String vs str"
 msgstr ""
 
-#: src/SUMMARY.md:33 src/basic-syntax/functions.md:1
+#: src/SUMMARY.md:34 src/basic-syntax/functions.md:1
 msgid "Functions"
 msgstr ""
 
-#: src/SUMMARY.md:34 src/basic-syntax/rustdoc.md:1
+#: src/SUMMARY.md:35 src/basic-syntax/rustdoc.md:1
 msgid "Rustdoc"
 msgstr ""
 
-#: src/SUMMARY.md:35 src/SUMMARY.md:82 src/basic-syntax/methods.md:1
+#: src/SUMMARY.md:36 src/SUMMARY.md:102 src/basic-syntax/methods.md:1
 #: src/methods.md:1
 msgid "Methods"
 msgstr ""
 
-#: src/SUMMARY.md:36
+#: src/SUMMARY.md:37
 msgid "Overloading"
 msgstr ""
 
-#: src/SUMMARY.md:37 src/SUMMARY.md:66 src/SUMMARY.md:90 src/SUMMARY.md:119
-#: src/SUMMARY.md:148 src/SUMMARY.md:177 src/SUMMARY.md:204 src/SUMMARY.md:225
-#: src/SUMMARY.md:251 src/SUMMARY.md:273 src/SUMMARY.md:293
+#: src/SUMMARY.md:38 src/SUMMARY.md:71 src/SUMMARY.md:105 src/SUMMARY.md:125
+#: src/SUMMARY.md:154 src/SUMMARY.md:184 src/SUMMARY.md:211 src/SUMMARY.md:232
+#: src/SUMMARY.md:260 src/SUMMARY.md:282 src/SUMMARY.md:303
 #: src/exercises/android/morning.md:1 src/exercises/bare-metal/morning.md:1
 #: src/exercises/bare-metal/afternoon.md:1
 #: src/exercises/concurrency/morning.md:1
 #: src/exercises/concurrency/afternoon.md:1
 msgid "Exercises"
 msgstr ""
 
-#: src/SUMMARY.md:38 src/exercises/day-1/implicit-conversions.md:1
+#: src/SUMMARY.md:39 src/exercises/day-1/implicit-conversions.md:1
 msgid "Implicit Conversions"
 msgstr ""
 
-#: src/SUMMARY.md:39
+#: src/SUMMARY.md:40
 msgid "Arrays and for Loops"
 msgstr ""
 
-#: src/SUMMARY.md:41
+#: src/SUMMARY.md:42
 msgid "Day 1: Afternoon"
+msgstr "第一天：午後"
+
+#: src/SUMMARY.md:44 src/SUMMARY.md:295 src/control-flow.md:1
+msgid "Control Flow"
+msgstr "控制流程"
+
+#: src/SUMMARY.md:45 src/control-flow/blocks.md:1
+msgid "Blocks"
+msgstr "區塊"
+
+#: src/SUMMARY.md:46
+msgid "if expressions"
+msgstr "if 表達式"
+
+#: src/SUMMARY.md:47
+msgid "for expressions"
+msgstr ""
+
+#: src/SUMMARY.md:48
+msgid "while expressions"
+msgstr ""
+
+#: src/SUMMARY.md:49
+msgid "break & continue"
+msgstr ""
+
+#: src/SUMMARY.md:50
+msgid "loop expressions"
 msgstr ""
 
-#: src/SUMMARY.md:43 src/basic-syntax/variables.md:1
+#: src/SUMMARY.md:52 src/basic-syntax/variables.md:1
 msgid "Variables"
 msgstr ""
 
-#: src/SUMMARY.md:44 src/basic-syntax/type-inference.md:1
+#: src/SUMMARY.md:53 src/basic-syntax/type-inference.md:1
 msgid "Type Inference"
 msgstr ""
 
-#: src/SUMMARY.md:45
+#: src/SUMMARY.md:54
 msgid "static & const"
 msgstr ""
 
-#: src/SUMMARY.md:46 src/basic-syntax/scopes-shadowing.md:1
+#: src/SUMMARY.md:55 src/basic-syntax/scopes-shadowing.md:1
 msgid "Scopes and Shadowing"
 msgstr ""
 
-#: src/SUMMARY.md:47 src/memory-management.md:1
+#: src/SUMMARY.md:56 src/enums.md:1
+msgid "Enums"
+msgstr "列舉"
```

Comment:
I am also not sure about this. But, according to my experiences reading some translated technical books, it's often not very clear what the translated terms mean without seeing the the originals. We could follow the example of the Traditional Chinese translation of the Rust Book, and create a terminology translation table at the end of the book.

Reply:
We will come back to this later.

Reply:
I think creating a glossary would be a good idea! It was discussed in #1184 as well.

Reply:
When zh-TW translation was first started, I think edong created [this glossary](https://docs.google.com/spreadsheets/d/1bVSJEUGkI9gcqmmA4va4y-I7LzgQUqNwnZYJsQNMBGE/edit#gid=168787290).  We also later found https://rust-lang.tw/book-tw/appendix-08-terminology.html

Reply:
That looks great! To get it into the course proper, we should add a `glossary.md` file somewhere near the end with a large table. That way the glossary will appear for all languages and it can become a joint resource.

Reply:
See #1277 for a draft of this. I don't know if this is helpful or if it just creates more noise?